### PR TITLE
fix: replace Haiku with Animator here and there

### PR DESCRIPTION
**Pull request by roperzh**
Summary of changes:

- Replace Haiku with Animator in a couple of missing places.

Regressions to look for:

- None expected

### CODE REVIEW

File: packages/haiku-creator/src/electron.js
Comment (zackbrown) [inline packages/haiku-creator/src/electron.js]: @roperzh is this (or the next one in this file) the title that will show up on macOS when you Cmd + Tab? Or is that the `productName` in haiku-creator/package.json?  (ideally that title is "Haiku Animator"; I'm just not sure which of these defines that, so you may already have it covered.)

**Comment by zackbrown** [comment]
Pending one question about whether it will say "Haiku Animator" when you command-tab (ideally yes), this LGTM.  Feel free to merge, @roperzh .

**Review by zackbrown — APPROVED** [review]

File: packages/haiku-creator/src/electron.js
Comment (stristr) [reply packages/haiku-creator/src/electron.js]: I'm pretty sure for the build, we need to change the `productName` in `package.json` (mono root).

**Comment by stristr** [comment]
Please do not merge. Noticed a few points of failure this morning.

**Comment by stristr** [comment]
LGTM, merge when ready.

**Comment by roperzh** [comment]
Thank you!

**Pull request merged**
